packages/theme/src/cli/flags.ts

```diff
@@ -31,3 +31,13 @@ export const themeFlags = {
     env: 'SHOPIFY_FLAG_ENVIRONMENT',
   }),
 }
+
+export const themeDevPreviewFlag = {
+  'dev-preview': Flags.boolean({
+    char: 'd',
+    required: false,
+    description: `Use the dev preview version of theme check
+Applies the typescript implementation of theme check to the theme`,
+    env: 'SHOPIFY_FLAG_DEV_PREVIEW',
+  }),
+}
```

Comment:
This was split into an exportable const so it may be used by the sibling PR which adds theme-check support to the CLI : https://github.com/Shopify/cli/pull/2889